src/options.h

```diff
@@ -635,6 +635,10 @@ sp_func_proto_paren;
 extern Option<iarf_e>
 sp_func_proto_paren_empty;
 
+// Add or remove space between function name and '(' with a typedef specifier.
```

Comment:
Suggestion: "...space between *alias name and '(' of a non-pointer function type typedef*." (emphasis to indicate changes).